addons/website_event_questions/models/event_question.py

```diff
@@ -26,7 +26,7 @@ class EventQuestion(models.Model):
     @api.constrains('event_type_id', 'event_id')
     def _constrains_event(self):
         if any(question.event_type_id and question.event_id for question in self):
-            raise UserError(_('Question cannot belong to both the event category and itself.'))
+            raise ValidationError(_("Question cannot be linked to both an Event and an Event Type."))
```

Comment:
Not a big deal for this but don't forget to re-export/complete the .pot when changing translated terms in stable.

Also minor correction for the commit message:

> Whether an question

Whether *a question

Reply:
Oh right, forgot to update that after replacing "object" with "question".

Just updated the pot as well.